samples/FunctionApp/Function3/Function3.cs

```diff
@@ -13,19 +13,26 @@ namespace FunctionApp
 {
     public static class Function3
     {
-
-        [FunctionName("Function3")]
-        [QueueOutput("name", "functionstesting2", Connection = "AzureWebJobsStorage")]
-        public static HttpResponseData Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequestData req,
+        [Function("Function3")]
+        public static MyOutputType Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", "post", Route = null)] HttpRequestData req,
             FunctionContext context)
         {
             var response = req.CreateResponse(HttpStatusCode.OK);
             response.WriteString("Success!");
 
-            context.OutputBindings["name"] = "some name";
-
-            return response;
+            return new MyOutputType()
+            {
+                Name = "some name",
+                HttpReponse = response
+            };
         }
     }
 
+    public class MyOutputType
+    {
+        [QueueOutput("functionstesting2", Connection = "AzureWebJobsStorage")]
```

Comment:
Weren't we going to make the name optional here?

Reply:
I thought we said, let's not have it all for now. And in future, if we need it, we can add an optional `Name` property. Let me know if you think otherwise.

Reply:
That's fine. Let's leave it as is.